### DISCUSSION TITLE
CR-10 SE PAUSE, RESUME, and M600

### DISCUSSION THREAD

**Discussion by nilrestforthewicken** [Guilouz/Creality-Helper-Script-Wiki#553] [Issues]
Hoping someone can help me out.  I need to be able to pause and resume.  Following several tutorials/repos, I have been unsuccessful.  I installed klipper via Creality Helper Script by Guilouz.  The PAUSE macro works, but it ignores the xy coordinates that I changed in the macro.  I changed this within the gcode_marco.cfg file.

Also, anytime I try to run RESUME macro I get the following error.

"{"code":"key165", "msg": "Error evaluating 'gcode_macro RESUME:gcode': jinja2.exceptions.UndefinedError: 'dict object' has no attribute 'fan2_speed' ", "values": ["gcode_macro RESUME:gcode", "jinja2.exceptions.UndefinedError: 'dict object' has no attribute 'fan2_speed' "]}"

Fan2 that it is referring to is only mentioned in the M600_support.cfg file which is located within the /Helper-Script directory.  It is locked and I am unable to edit that cfg.

M600 has been redirected to the PAUSE macro, but still only pauses the printhead right where it was when the PAUSE command was sent, and doesn't park it at the coordinates that I specified.

I have been stuck on this issues for a week now.

**Comment by Guilouz** [comment]
M600_support.cfg file Is only for K1 series.
Script is not compatible sith CR10 SE beacaise I don’t have it.
I can only support that I have tested.

You error means there is an issue with fan2 variable, CR10 SE have a fan2?

**Reply by nilrestforthewicken** [reply]
Thank you for responding!  I apologize for posting my issue here since you haven't been involved with the CR-10 SE adaptation of your work/script.  I finally managed to fix it after constantly seeing the fan2 error, I decided to uninstall the M600_support.cfg through the helper.  Once, I did that, it resolved my issue.  I left the M600 alias as PAUSE and everything works as it should.  Thank you for all of your work with this script.  It made the installation a breeze.  Now, I just need to dive into MACROs a bit more, and learn the terms/references that are used in the cfg files.
